src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptEditorInlineRenameService.cs

```diff
@@ -22,6 +24,13 @@ internal sealed class VSTypeScriptEditorInlineRenameService(
 {
     private readonly Lazy<VSTypeScriptEditorInlineRenameServiceImplementation>? _service = service;
 
+    public bool IsRenameContextSupported => false;
+
+    public Task<ImmutableDictionary<string, ImmutableArray<string>>> GetRenameContextAsync(IInlineRenameInfo inlineRenameInfo, IInlineRenameLocationSet inlineRenameLocationSet, CancellationToken cancellationToken)
```

Comment:
note: my preference would be that we have a *default* impl that finds the defs/references and includes their line (or a few lines around them.  That way all languages would pick this up.  *Then* a lang like C# ca say "oh, i want to customize this to get surrounding method blocks".